ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -809,6 +809,24 @@ tests:
         AGENT_USE_APPLIANCE_MODEL=true
     workflow: agent-e2e-compact-ipv4
   timeout: 6h0m0s
+- as: e2e-agent-compact-ipv4-appliance-diskimage
+  cluster: build05
+  optional: true
+  run_if_changed: ^(cmd/.*|data/.*|pkg/.*|Dockerfile\..*|Makefile|go\.mod|go\.sum)$
```

Comment:
Any specific reason for not using the same regex of the other agent jobs?

Reply:
Not really... Changed to: /agent/